src/octoprint/plugins/corewizard/templates/corewizard_onlinecheck_wizard.jinja2

```diff
@@ -11,7 +11,7 @@
     OctoPrint comes preconfigured to perform the connectivity check every 15 minutes. You may change the value here.
 {% endtrans %}</p>
 
-<form class="form-horizontal" data-bind="with: settingsViewModel, enable: !setup(), css {disabled: setup()}" onsubmit="return false;">
```

Comment:
These enable/css properties on the forms do not appear to work. I removed them entirely because I don't see any reason why they should be disabled regardless of the choice to enable/disable the top-level feature.

Reply:
I honestly can't tell you what this was supposed to accomplish either, so yeah, good riddance.